lib/proofer/vendor/mock.rb

```diff
@@ -13,6 +13,10 @@ class Mock < VendorBase
         'bear'  => 'schools'
       }.freeze
 
+      SUPPORTED_STATES = %w(
+        WA ID CA NM TX ND SD NE IA AR WI IL MS MI IN KY FL PA VA MD DC DE NA ME AZ
```

Comment:
Where did this list come from? Is there a reference we can link to in a comment?

Also, I think it would be easier to scan or search if it were alphabetized. Maybe not necessary though if it's pretty much never going to change.

Reply:
Actually, we may be adding states as we go. Agree with Tim that this list may work a bit better for us if it is alphabetized.

Reply:
I believe this list is in the AAMVA docs. It's private so I am not sure we can link it to here. Agreed about alphabetizing, I'll do that 👍

Reply:
I see what you're saying about not being linkable, I'm just trying to pre-empt questions around why we have this subset of states and where they came from. Perhaps just a note that this should match with AAMVA?  Not sure what we can do here.

Reply:
That makes sense, but this is just a mock proofer and I know Jonathan is working on the AAMVA specific proofer. I believe this is generalized to how all our other proofing channels would work, but I might be wrong maybe @jmhooper knows?